website/docs/plugin/which-sdk.mdx

```diff
@@ -48,12 +48,8 @@ should consider using the framework if you would benefit from the ability to:
 ## Do You Need Any Features That the Framework Doesn't Provide (Yet)?
 
 The framework is still under development, and so it may not yet support some
-features that are available in SDKv2. These include the ability to:
-
-* Use timeouts.
-
-These features are on our roadmap to implement and support, but if you need
-them _today_, SDKv2 may be a better choice.
+features that are available in SDKv2. If this is the case, SDKv2 may be a
```

Comment:
```suggestion
features that are available in SDKv2. In that case, SDKv2 may be a
```

Suggest changing "If this is the case" to "In that case" to avoid "This" without a subject.